src/brewHandler.h

```diff
@@ -31,16 +31,13 @@ enum BackflushState {
 };
 
 // Normal Brew
-BrewState brewCounter = kBrewIdle;
-int brewSwitch = 0;
-int brewSwitchTrigger = LOW;
-int lastStateBrewTrigger;                        // the last valid reading from the input pin (debounced)
-unsigned long lastDebounceTimeBrewTrigger = 0;  // the last time the output pin was toggled
-unsigned long debounceDelayBrewTrigger = 20;    // >20ms when switches are "flicked"
-unsigned long brewSwitchTriggerMillis = 0;
-const unsigned long brewTriggerLongPress = 500;     // time in ms until brew trigger will be interpreted as manual brewing
-int brewSwitchTriggerCase = 10;
+BrewState currBrewState = kBrewIdle;
+
+uint8_t currStateBrewSwitch = LOW;
+uint8_t currBrewSwitchStateMomentary = LOW;
```

Comment:
These variables sound very similar, but it's not clear from their names (especially the second one) what they're for

Reply:
Why do we need two variables anyway though, the switch can always only have one type I guess

Reply:
To be perfectly honest, I didn't change much about the logic of how the momentary switch is handled. My main goal was to handle the debouncing in the Switch class and make the variable names a bit more consistent/less confusing.

We should refactor that in a followup PR because it's working at the moment and I don't want to get he HAL into master as soon as possible.

Reply:
Brewswitch state mashine and brew state mashine should get a refactore, yes
But as you said, in another PR. I did put it on the roadmap.

Reply:
Sure, my second comment was a note on the side. But the variable names could be improved, they are part of the PR.

Reply:
I think this would be an easy change that could be done before the merge